src/components/ItaliaTheme/Blocks/Listing/BandiInEvidenceTemplate.jsx

```diff
@@ -201,6 +201,17 @@ const BandiInEvidenceTemplate = ({
                         </span>
                       </span>
                     )}
+
+                    {/* Note aggiornamenti */}
+                    {item.update_note &&
+                      (item.bando_state?.includes('open') ||
+                        item.bando_state?.includes('inProgress')) && (
+                        <span className="d-flex flex-wrap align-items-baseline bando-dati-info">
+                          <span className="bando-dati-date">
```

Comment:
questa classe riferita alle date è voluta? 
ne farei una specifica per le note e al massimo gli assegnerei gli stessi stili della data

Reply:
per il resto va bene

Reply:
no in effetti non servivano, ho fatto pulizia